assets/app/scripts/controllers/build.js

```diff
@@ -69,6 +69,7 @@ angular.module('openshiftConsole')
                 };
               }
               $scope.build = build;
+              $scope.logCanRun = !(_.includes(['New', 'Pending'], build.status.phase));
```

Comment:
Yeah, I think a message saying there is no log, an error prevented the build from starting.